src/services/LocalTime.tsx

```diff
@@ -0,0 +1,34 @@
+import config from "../config/config.json";
+import { Worker } from "../moudels/Worker";
+
+export const fetchLocalTime = async (worker: Worker): Promise<Worker> => {
+  const X_Api_Key: string | undefined = import.meta.env.VITE_Api_Key;
```

Comment:
X_API_KEY - if that is the name you decided to use - consts are all caps.
You could also change the name to something clearer if you would like